### PR TITLE
Pull request #1: Nav and header for mobile version of personal portfolio

### PR DESCRIPTION
**Pull request by Anjacodes**
In this step of the project I:

* Built the HTML mobile version of the skeleton structure for my portfolio
* Included linters for HTML/CSS
* Added a stylesheet to match the portfolio template 3 on Figma
* Added a descriptive README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page. Show your support
+
+Give a ⭐️ if you like this project! Acknowledgments
+
+📝 License
+
+This project is MIT licensed.
```
Comment (Mihndim2020) [inline README.md:31]: - [ ] Please make sure you follow the microverse readme [template](https://github.com/microverseinc/readme-template) and fill out the available information related to your project. Some vital information like; step to set up your project locally, link the license, acknowledgments etc is missing. Please also make sure the sub-headings are formatted appropriately as it makes your readme file more presentable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </nav>
```
Comment (Mihndim2020) [inline index.html:18]: - [ ] The container of this section should be a `<header>` tag. Please kindly fix that as this makes your html more [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
```
Comment (Mihndim2020) [inline index.html:15]: - [ ] If this is a logo, then it should be be a link(`<a>`), if you notice on most websites, the logo is usually a link that takes you to the home page. please kindly fix that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </nav>
+
+      <!-- Header -->
+      <header>
```
Comment (Mihndim2020) [inline index.html:21]: - [ ] The container of this section should be a `<section`> tag as this makes your html more [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp). Please kindly fix that.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <section class="header_container">
+          <h1 class="heading1">Hey there. I’m Anja.</h1>
+          <h1 class="heading2">I’m a software developer</h1>
```
Comment (Mihndim2020) [inline index.html:24]: - [ ] Your page is suppose to have only one `<h1>` tag as this make you html semantic, helps SEO, and makes your site more accessible to assistive technologies.

Comment (Mihndim2020) [inline index.html:24]: - [ ] This should be a `<p>` tag, please kindly fix that as it makes your html more semantic.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+              rel="noopener noreferrer"
+              ><img src="images/github.svg" alt="Github"
+            /></a>
+            <a
+              href="https://twitter.com/Anja_Schmidt7"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="images/Linkedin-icon.svg" alt="linkedin"
+            /></a>
+          </div>
```
Comment (Mihndim2020) [inline index.html:49]: - [ ] Please kindly put this lists of icons in a `<ul>` tag to group them, `<li>` tag per icon, each icon should have a link (`<a>` tag). Kindly fix that,  as this makes your html more semantic.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Anjacodes, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Good design implementation
:heavy_check_mark: Linters are passing

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [ ] It is a good thing you are using imperative tense for your commit messages, please make sure all first letters for the first words are capitalized as this makes your commit messages clean and professional. (***this is for your next commit messages***)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Anjacodes** [comment]
Hi Mihndim, 

Thank you for your comments and for reviewing this so quickly! 
I will be sure to implement all your suggestions during the allocated time slot first thing tomorrow morning. 

Wishing you a great day further!
Best,
Anja 


⁣Gesendet mit BlueMail ​

Am 19. Okt. 2021, 19:46, um 19:46, Mihndim Julius ***@***.***> schrieb:
***@***.*** requested changes on this pull request.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <header>
+        <ul>
+          <li><a href="#" rel="noopener noreferrer">Welcome</a></li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </header>
```
Comment (mikethreels) [inline index.html:18]: - [ ] Good job on implementing the `<header>` tag 👍 however the `<nav>` tag should still be used for this part of the code, it should look something like below example.
```
<header>
  <nav>
    ...
  </nav>
<header>
```

File: index.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+      <!-- Header -->
+
+      <section id="main">
+        <section class="header_container">
+          <h1 class="heading heading1">Hey there. I’m Anja.</h1>
+          <p class="heading heading2">I’m a software developer</p>
```
Comment (mikethreels) [inline index.html:25]: - [ ] Good job removing the double `<h1>` as the previous CR mentioned. 👍 however according to the requirements the subtitle should still be a header tag `hx (examples: <h2>,<h3>, etc...)`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
```
Comment (D4nielJ) [inline index.html:11]: [OPTIONAL] I want to recommend you to not use this container, instead, use the dev tools from your web browser to resize the window of your web app when you're working on the mobile version. Your page will automatically resize to any viewport and in upcoming lessons, you'll learn how to modify its behavior based on the width of the window.

I hope you find this useful! :slightly_smiling_face: 

image

This is a preview of how you can use the dev tools to resize the viewport.

**Review by D4nielJ — APPROVED** [review]
## Hello @Anjacodes, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**